cmd/cmd/remote.go

```diff
@@ -26,8 +26,8 @@ import (
 )
 
 const remoteCmdLiteral = "remote"
-const remoteCmdShortDesc = "Add, remove, update or select Micro Integrator"
-const remoteCmdLongDesc = "Add, remove, update or select Micro Integrator which will be associated with the CLI\n"
+const remoteCmdShortDesc = "Add, remove, update or select Micro Integrator, login or logout of Micro Integrator"
```

Comment:
How about, "Add, login to, logout of, remove, update or select Micro Integrator"

Reply:
Fixed in 2514a25d799cfe90b35dd0fdda17773b127eb19c